### PR TITLE
install ipympl

### PR DESCRIPTION
**Pull request by djherbis**
Adds widget support for matplotlib

http://b/185060120

### CODE REVIEW

**Comment by djherbis** [comment]
Needed for https://github.com/Kaggle/kaggleazure/pull/15395 to work out of the box on our env

File: Dockerfile [hunk 1]
```diff
@@ -422,6 +422,8 @@ RUN pip install flashtext && \
     pip install easyocr && \
     # Keep JAX version in sync with GPU image.
     pip install jax==0.2.12 jaxlib==0.1.64 && \
+    # ipympl adds interactive widget support for matplotlib
+    pip install ipympl && \
```
Comment (rosbo) [inline Dockerfile:426]: Probably hard to add a test for this right?

**Review by rosbo — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (djherbis) [reply Dockerfile:426]: Yeah I don't think theres a way to test an interactive widget like this :(

Comment (djherbis) [reply Dockerfile]: Adding a version pin to make sure it stays compatible

**Pull request merged**